examples/04-Exporting-ranking-models.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<p>Can the output_path not be based on DATA_FOLDER?</p><p> </p><p>output_path = os.path.join(DATA_FOLDER, 'processed/')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>